ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -48,11 +48,6 @@ tests:
   commands: RUN_ORIGIN_TESTS=true make functests-on-ci
   openshift_installer_src:
     cluster_profile: gcp
-- as: cnf-e2e-gcp-periodic
-  commands: make functests-on-ci
```

Comment:
what's the alternative?

Reply:
We remove the job from the config because we are detaching it from the config.
It's the only way to be able to specify the cluster variant, otherwise the CI will complain about the discrepancy between the config and the job / will try to override the job.

Reply:
so we lose the cron job? this helped us to find issues in the past when there was a change in openshift that affect CNF

Reply:
No, it's there. It's just detached from the configuration, because in the config you can't specify the ovn variant.
If you look at the job files, the job is still there.

Reply:
ok thanks then it looks good :)